### PR TITLE
Change label: 'Allow Download' to 'Allow Download/Sync' for all the languages.

### LINKED ISSUE #46700: Change label: "Allow Download"
This is only a small thing: When I share a folder there is the option to allow downloads. In German "Download erlauben".

I thought, this only applies for downloads from the browser. But it also disables half the sync function if you use the virtual file system. You can only see all folders and files, but you can't open them. The desktop client shows a complicated "Forbidden"-error message.

1. It would help if the label said "Allow Download/Sync" - "Download/Sync erlauben". 
2. It would be cool, if the error message in the desktop client was easier to understand: "Download/Sync disabled"

### PR DESCRIPTION
**Pull request by Agnibaan**
* Resolves: #46700

## Summary

         changed "Allow Download" label to "Allow Download/Sync" in shared folder page for better understanding . Its includes all the language change not only german and english.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by Agnibaan** [comment]
@joshtrichards can you review this PR?

**Comment by AndyScherzinger** [comment]
@nextcloud/designers for feedback on the label-change
@Agnibaan please remove the l10n changes since they will be auto-synced back and forth between the Github repository and transifex- so they get overridden every 24h

**Comment by Agnibaan** [comment]
> @nextcloud/designers for feedback on the label-change @Agnibaan please remove the l10n changes since they will be auto-synced back and forth between the Github repository and transifex- so they get overridden every 24h

@AndyScherzinger  i think the issue resolved now , i reversed the changes as suggested . you can review it.

**Comment by AndyScherzinger** [comment]
@Agnibaan see https://github.com/nextcloud/server/pull/47093/files -> 85 files changed -> so the files in the l10n are still there in terms of changeset and needs to be reverted

**Comment by Agnibaan** [comment]
> @Agnibaan see https://github.com/nextcloud/server/pull/47093/files -> 85 files changed -> so the files in the l10n are still there in terms of changeset and needs to be reverted

u want me to reverse all the changes in all the languages? then i have to rebase this branch completely, no issue i will do it just confirm me if I m right?

**Comment by AndyScherzinger** [comment]
@Agnibaan let'S first get a confirmation from @jancborchardt about the label change in general. If he accept the change than yes, all files changes within l10n should be reverted since the bot will override them back to what they had been anyways.

File: apps/files_sharing/src/views/SharingDetailsTab.vue [hunk 1]
```diff
@@ -154,7 +154,7 @@
 						:disabled="!canSetDownload"
 						:checked.sync="canDownload"
 						data-cy-files-sharing-share-permissions-checkbox="download">
-						{{ t('files_sharing', 'Allow download') }}
+						{{ t('files_sharing', 'Allow Download/Sync') }}
```
Comment (susnux) [inline apps/files_sharing/src/views/SharingDetailsTab.vue:157]: Only this is needed, the rest will be overwritten by Transifex as the translation is handled there.

Comment (susnux) [inline apps/files_sharing/src/views/SharingDetailsTab.vue:157]: Also the casing is wrong, either:
```suggestion
						{{ t('files_sharing', 'Allow download / sync') }}
```

or

```suggestion
						{{ t('files_sharing', 'Allow download and sync') }}
```

Comment (Agnibaan) [reply apps/files_sharing/src/views/SharingDetailsTab.vue]: updated as suggested.

Comment (Agnibaan) [reply apps/files_sharing/src/views/SharingDetailsTab.vue]: okay got it!!

**Review by nickvergessen — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Comment by Agnibaan** [comment]
@susnux 
why not merging this request ? anything wrong?

**Comment by nickvergessen** [comment]
yeah, CI is not green.
I will send the PR from "not a fork", as I'm not sure this PR here would actually touch/break cypress tests

**Comment by nickvergessen** [comment]
Local PR at https://github.com/nextcloud/server/pull/47198

**Comment by nickvergessen** [comment]
@Agnibaan thanks for your PR, merged it via the one with green CI above

**Pull request closed without merging**